### PR TITLE
Review and update find command

### LINKED ISSUE #143: Review and update find functionality

### PR DESCRIPTION
**Pull request by KennethTeo2002**
Closes #143 

what's added:
- partial name search functionality.

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -378,6 +378,15 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
     Use case ends.
 
+**Use case: UC5 - Find people**
+
+**MSS**
+
+1.  Actor requests to find specific people.
```
Comment (sheenkerr) [inline docs/DeveloperGuide.md:385]: Maybe can add an extension for no person found?

**Review by sheenkerr — CHANGES_REQUESTED** [review]

**Review by sheenkerr — DISMISSED** [review]
LGTM

**Review by joeltio — APPROVED** [review]
LGTM

**Review by ITLimJiaWei — APPROVED** [review]
LGTM

**Pull request merged**